pkg/registry/policy/poddisruptionbudget/strategy.go

```diff
@@ -109,7 +110,9 @@ func (podDisruptionBudgetStatusStrategy) PrepareForUpdate(ctx context.Context, o
 
 // ValidateUpdate is the default update validation for an end user updating status
 func (podDisruptionBudgetStatusStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
-	// TODO: Validate status updates.
-	return field.ErrorList{}
-	// return validation.ValidatePodDisruptionBudgetStatusUpdate(obj.(*policy.PodDisruptionBudget), old.(*policy.PodDisruptionBudget))
+	var version string
+	if requestInfo, found := genericapirequest.RequestInfoFrom(ctx); found {
+		version = requestInfo.APIVersion
```

Comment:
construct a schema.GroupVersion parameter to pass to validation using both the APIVersion and APIGroup